modules/serverless-creating-subscriptions.adoc

```diff
@@ -56,5 +59,19 @@ spec:
 . Apply the YAML file by entering:
 +
 ----
-$ oc apply -f <FILENAME>
+$ oc apply -f <filename>
+----
+
+== Create a subscription by using the `kn` CLI
+
+.Procedure
+
+. Create a Subscription object to connect a channel to a service, by entering the following command:
++
+
+[source,terminal]
+----
+$ kn subscription create <subscription-name> --channel <channel-name> --mysink <type:sink-name>
 ----
+// ksvc:mysvc
+// QUESTION: Are there different types besides ksvc here that can be a sink? How does format change e.g. if you use another resource, e.g. another channel?
```

Comment:
We have the following built-in channels:

* `ksvc:` or no prefix --> Knative service
* `broker:` --> connect to a broker
* `http://` --> connect to an arbitrary URL (can also be https)

Actually, it the same sink types that you can use when attaching a source to a sink or connecting a trigger to a sink (also, you can use a custom mapping in the configuration file, but I'm not sure if we already document that).

@navidshaikh is this explanation correct ?

Reply:
I am finalizing the the subscription CRUD on kn CLI, give me a couple days to comeback on this with proper flags and mapping.